casr/src/util.rs

```diff
@@ -293,3 +284,532 @@ pub fn get_output(command: &mut Command, timeout: u64, error_on_timeout: bool) -
             .with_context(|| format!("Couldn't launch {command:?}"))
     }
 }
+
+/// Create output, timeout and oom directories
+///
+/// # Arguments
+///
+/// `matches` - tool argumnets
+///
+/// Return value
+///
+/// Paths to (output, oom, timeout) directories
+fn initialize_dirs(matches: &clap::ArgMatches) -> Result<(&PathBuf, PathBuf, PathBuf)> {
+    // Get output dir
+    let output_dir = matches.get_one::<PathBuf>("output").unwrap();
+    if !output_dir.exists() {
+        fs::create_dir_all(output_dir).with_context(|| {
+            format!("Couldn't create output directory {}", output_dir.display())
+        })?;
+    } else if output_dir.read_dir()?.next().is_some() {
+        bail!("Output directory is not empty.");
```

Comment:
May be it will be usefull to add `--force` flag for ignoring this error? (I really miss such option)

Reply:
To all casr tools with some sort of out dir

Reply:
Perhaps, but I think it needs to be tested separately and not in this pull